flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/StreamExecutionEnvironment.scala

```diff
@@ -124,7 +125,7 @@ class StreamExecutionEnvironment(javaEnv: JavaEnv) {
    * Gets the checkpoint config, which defines values like checkpoint interval, delay between
    * checkpoints, etc.
    */
-  def getCheckpointConfig = javaEnv.getCheckpointConfig()
```

Comment:
http://stackoverflow.com/questions/8124055/scala-tostring-parenthesize-or-not   http://stackoverflow.com/questions/12334936/why-does-scala-need-parameterless-in-addition-to-zero-parameter-methods